AEPAssurance/Source/QuickConnect/QuickConnectView.swift

```diff
@@ -403,7 +403,7 @@ class QuickConnectView: SessionAuthorizingUI {
     
     func sessionConnected() {
         self.connectionSuccessfulState()
-        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+        DispatchQueue.main.async {
```

Comment:
I am curious why we had this 1-second delay in the first place. Is it to show some animation?

Reply:
There is an animation, but it happens in the block. So it still doesn't make sense.